### PR TITLE
Sort insights table total sum

### PR DESCRIPTION
**Pull request by liyiy**
## Changes

*Please describe.*  
*If this affects the frontend, include screenshots.*  

Adds default sort order and sorting to insights table breakdown
Addresses #5383 

https://user-images.githubusercontent.com/25164963/128783977-1f32d3ce-97a5-46d9-a74a-e02a29bb46f3.mov

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] New/changed UI is decent on smartphones (viewport width around 360px)

### CODE REVIEW

File: frontend/src/scenes/insights/InsightsTable/InsightsTable.tsx [hunk 1]
```diff
@@ -164,10 +164,7 @@ export function InsightsTable({ isLegend = true, showTotalCount = false }: Insig
                         filters.display === ACTIONS_PIE_CHART)
                 ) {
                     // Special handling because `count` will contain the last amount, instead of the cumulative sum.
```
Comment (liyiy) [inline frontend/src/scenes/insights/InsightsTable/InsightsTable.tsx:166]: @paolodamico Do you know when count shows last amount over cumulative sum? It seems like it's passing cumulative sum now, but want to double check 😄

Comment (paolodamico) [reply frontend/src/scenes/insights/InsightsTable/InsightsTable.tsx]: Can't remember from the top of my head when this happens, tried a bunch of different things and couldn't find this happening so I think it's safe to remove. Maybe worth just checking with Core Analytics as a precaution.

**Review by paolodamico — CHANGES_REQUESTED** [review]
Found two UX bugs worth addressing.

1. The column header now changes to two lines (but only in some cases which causes confusion)
image

2. Clicking on the arrow to show the dropdown also changes the ordering.
    image

**Comment by liyiy** [comment]
@paolodamico I've already pushed the changes but it seems it hasn't updated on github's end yet, I've changed the dropdown click to a hover instead to prevent the selection + sorting issue.

https://user-images.githubusercontent.com/25164963/128904300-fac9d72e-94fe-426a-8aa6-1a80beaf6c55.mov

**Review by paolodamico — APPROVED** [review]
The hover on the dropdown feels a bit off/cumbersome, but I think is good enough for now!

**Pull request merged**